packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -146,6 +177,9 @@ export default {
 
             resolveCustomerDetails(data, state);
 
+            // This call can be removed when newly created guest JWT token has phone number claim populated
+            await enrichPhoneNumber(data.customer, state, getCustomerUrl, timeout);
```

Comment:
would this need to be after line 183 as you haven't updated the state yet? Then you could also remove the data.customer part and just use what's in the stae

Reply:
@FayeCarter no, it uses state in the same manner that resolveCustomerDetails uses it, just to retrieve the authToken for the api call

Reply:
@FayeCarter the second bit with customer being there on the state, yes

Reply:
Refactored